src/main/java/com/redhat/documentation/asciidoc/extraction/SourceExtractor.java

```diff
@@ -30,7 +30,10 @@
         EXAMPLE("===="),
         SOURCE("----"),
         PARAGRAPH(""),
-        LISTING("----");
+        LISTING("----"),
+        IMAGE("image::"),
```

Comment:
pretty sure IMAGE and THEMATIC_BREAK are not proper defined this way - but just added it as a way to avoid error thrown :)

Reply:
Cool, didn't think about images :) Good catch

Reply:
Looks like we need to special case image. Not sure what a thematic break is.

Reply:
Looks like a page break is the same.

Reply:
This is now included